Cargo.toml

```diff
@@ -129,6 +129,7 @@ tokio              = { version = "1.38.0", default-features = false }
 tracing            = "0.1.40"
 tracing-chrome     = "0.7.2"
 tracing-subscriber = "0.3.18"
+urlencoding        = "2.1.3"
```

Comment:
What's this for?

Reply:
The urlencoding crate is added for the SVG format, taking <script> for example, it will transform it into %3Cscript%3E, which is used in some <-included SVG files. Without this crate, the compilation of official icon of rolldown https://rolldown.rs/lightning-down.svg will raise errors.